File: TransCoder_tokenized_test_set_functions/java/PROGRAM_CHECK_WATER_TANK_OVERFLOWS_N_SOLID_BALLS_DIPPED_WATER_TANK.java
static void overflow ( int H, int r, int h, int N, int R ) {
  double tank_cap = 3.14 * r * r * H;
  double water_vol = 3.14 * r * r * h;
  double balls_vol = N * ( 4 / 3 ) * 3.14 * R * R * R;
  double vol = water_vol + balls_vol;
  if ( vol > tank_cap ) {
    System . out . println ( "Overflow" );
  }
  else {
    System . out . println ( "Not in overflow state" );
  }
}

File: TransCoder_tokenized_test_set_functions/cpp/PROGRAM_CHECK_WATER_TANK_OVERFLOWS_N_SOLID_BALLS_DIPPED_WATER_TANK.cpp
void overflow ( int H, int r, int h, int N, int R ) {
  float tank_cap = 3.14 * r * r * H;
  float water_vol = 3.14 * r * r * h;
  float balls_vol = N * ( 4 / 3 ) * 3.14 * R * R * R;
  float vol = water_vol + balls_vol;
  if ( vol > tank_cap ) {
    cout << "Overflow" << endl;
  }
  else {
    cout << "Not in overflow state" << endl;
  }
}